### ISSUE TITLE
Issue with Rails 6.0.0.rc1

### ISSUE THREAD

**Issue by nadiyaka** [salsify/goldiloader#87]
I have received many duplicated errors associated with Goldiloader after Rails update to 6.0.0.rc1 version.

```
NoMethodError:
       undefined method `set_value' for #<ActiveRecord::AssociationRelation []>
       Did you mean?  offset_value
     # .../ruby-2.6.2/gems/goldiloader-3.1.0/lib/goldiloader/active_record_patches.rb:63:in `auto_include_value='
     # .../ruby-2.6.2/gems/goldiloader-3.1.0/lib/goldiloader/active_record_patches.rb:73:in `merge_single_values'
```

Can you please help me to fix them?

**Comment by jturkel** [comment]
@nadiyaka - Can you confirm this issue is fixed in Goldiloader 3.1.1?

**Comment by nadiyaka** [comment]
> @nadiyaka - Can you confirm this issue is fixed in Goldiloader 3.1.1?

Yes! Thank you :)
